.travis.yml

```diff
@@ -8,7 +8,8 @@ matrix:
     -   env: TOXENV=py36
         python: 3.6
     -   env: TOXENV=pypy
-install: pip install coveralls tox
+#install: pip install coveralls tox
```

Comment:
note that the line you actually want to ax is like 17 `after_success: coveralls`